stack-deploy.cabal

```diff
@@ -71,7 +71,7 @@ library
     , optparse-applicative >=0.14 && >=0.15 && <0.16
     , random >=1.1 && <1.2
     , source-constraints >=0.0.1 && <0.1
-    , stratosphere >=0.59 && <0.60
+    , stratosphere >=0.57 && <1
```

Comment:
@epicallan Is this a hand edit? I do not see the corresponding `package.yaml` change.

Reply:
Sorry, yes it is. Let me fix